api/searchQueue/index.js

```diff
@@ -112,6 +112,15 @@ exports.handler = require("leo-sdk/wrappers/resource")(async (event, context, ca
     })).on("finish", () => {
         clearTimeout(fullTimeout);
         clearTimeout(timeout);
-        callback(null, response)
+        //console.log(queue, "total size:", size, response.results.length);
+        callback(null, {
+            statusCode: 200,
+            headers: {
+                'content-encoding': "gzip",
+                'Content-Type': 'application/json',
+            },
+            body: require("zlib").gzipSync(JSON.stringify(response)).toString("base64"),
```

Comment:
@czirker Do you want to check to see if the caller can accept a gzip response first? Or are we just assuming they can because this is an API specifically for Botmon? I'm good either way, just wanted to know your thoughts.

Reply:
for this one it is just botmon so i was just assuming it accepts gzip